lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -116,9 +116,7 @@ define([
         showErrorMessage: function (message) {
             this.$('.wrapper-msg')
                 .removeClass('is-hidden')
-                .find('.msg-content .copy').text(message)
-                .closest('.msg')
-                .focus();
```

Comment:
Yes, this was causing focus on just the paragraph message, which caused a weird display issue. I tried to adjust focus to the entire ``.wrapper-msg`` element using both the ``.parent()`` and ``.closest()`` JQuery methods, but it seemed not to work. The idea was to remove this for now until we can A) do a larger a11y audit with @cptvitamin or B) move focus to the ``.wrapper-msg`` element to resolve the visual display issue.

For the time being, I'm happy to revert this back to the way it was if @cptvitamin thinks its a regression.